src/Modal/Modal.spec.tsx

```diff
@@ -165,3 +167,39 @@ test("when content area is clicked, `onClose` should not be called", () => {
 
   expect(onClose).not.toHaveBeenCalled();
 });
+
+test("when passed `as` prop, props should be merged", () => {
+  const testId = faker.lorem.word();
```

Comment:
To clarify: the purpose is that this is a value and we don't care what it is. Anything coming from `faker` is DNC (do not care) and everywhere else is presumably important.

I suspect the mental overhead is because we're not familiar with faker (I had to look at the typescript definitions to know what I should use), and that using it more will remove that mental overhead.

That all being said, I think I want to keep it this way.